inst/json/graph_cost_efficacy_config.json

```diff
@@ -1,81 +1,87 @@
 {
         "metadata": {
-                "x_col": "efficacy",
-                "y_col": "cost",
+                "x_col": "reductionInCases",
                 "id_col": "intervention",
-                "format": "long"
+                "format": "long",
+                "settings": ["irsUse", "netUse"]
         },
         "series": [
             {
                 "id": "none",
+                "y_formula": ["({population} / {procurePeoplePerNet}) * {priceNetStandard}"],
                 "name": "No intervention",
                 "type": "scatter",
                 "marker": {"color": "grey", "size": 10},
                 "error_x": {
                     "type": "data",
                     "width": 0,
-                    "col": "efficacyErrorPlus",
-                    "colminus": "efficacyErrorMinus"
+                    "col": "reductionInCasesErrorPlus",
+                    "colminus": "reductionInCasesErrorMinus"
                 }
             },
             {
                 "id": "ITN",
+                "y_formula": ["({population} / {procurePeoplePerNet}) * {priceNetStandard}  + ({priceDelivery} * {population} * (({procureBuffer} + 100)/ 100))"],
```

Comment:
Should this be `yFormula`?

Reply:
For now the Vue schemas are using camelCase, ticket to sort those out in one go: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-1878

Reply:
snak case :hamburger:\_:fries:\_:candy: